js/ui/components/clubhouse.js

```diff
@@ -207,9 +118,10 @@ var ClubhouseAnimation = class ClubhouseAnimation extends Animation {
 };
 
 var ClubhouseAnimator = class ClubhouseAnimator {
-    constructor(proxy) {
+    constructor(proxy, clubhouseId) {
         this._proxy = proxy;
         this._animations = {};
+        this._clubhouseId = clubhouseId;
```

Comment:
I also think that you can use here just an argument isOldClubhouse. Or something similar.